sdk/inc/azure/core/az_result.h

```diff
@@ -60,13 +60,13 @@ enum
     } \
   } while (0)
 
-/**
- * The type represents success and error conditions.
- * Bits:
- * - 31 Severity (0 - success, 1 - failure).
- * - 16..30 Facility.
- * - 0..15 Code.
- */
+///
+/// The type represents success and error conditions.
```

Comment:
You need `///` for the `The type represents success and error conditions.` sentence, but use `//` later on for the bits part.
`///` comments still go to Doxygen - the same as `/** */`.

Reply:
I.e. I propose this:
```
 /// The type represents success and error conditions.
// Bits:
// - 31 Severity (0 - success, 1 - failure).
// - 16..30 Facility.
// - 0..15 Code.
```